Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -0,0 +1,139 @@
+# Getting started with the Mixed Reality Toolkit
+
+![](/External/ReadMeImages/MRTK_Logo_Rev.png)
+
+The Mixed Reality Toolkit (MRTK) is a new platform for building your Mixed Reality experiences for Virtual Reality (VR) and Augmented Reality (AR), aimed at accelerating your content production by removing any of the common complexities that complicate projects in a true multi-platform / multi-vr solution.
+The MRTK solves this by providing a common logical platform to build your dream, together with a rich set of services and SDK components for building your project.  Al these components are geared towards the new MRTK platform that abstracts out all the different ways that each platform delivers its experience.
+
+## Prerequisites
+
+To get started with the Mixed Reality Toolkit you will need:
+
+* [Unity 2018.2.13f1 +](https://unity3d.com/get-unity/update)
```

Comment:
Stabilization branch is on 14

Reply:
I know, but I stated the MINIMUM :D 
Hence the +!!